cmd/collection/main.go

```diff
@@ -127,48 +127,48 @@ func main() {
 				"the delay to broadcast block proposal in order to control block production rate")
 		}).
 		Initialize().
-		Module("mutable follower state", func(node *cmd.FlowNodeBuilder) error {
+		Module("mutable follower state", func(node cmd.NodeBuilder) error {
```

Comment:
All `main.go` now use the interface `cmd.NodeBuilder` instead of the actual implementation `FlowNodeBuilder`